R/github-labels.R

```diff
@@ -81,11 +100,17 @@ use_github_labels <- function(delete_default = FALSE,
   }
 }
 
-gh_labels <- c(
-  "bug" =         "e02a2a",
-  "feature" =     "009800",
-  "reprex" =      "eb6420",
-  "wip" =         "eb6420",
-  "docs" =        "0052cc",
-  "performance" = "fbca04"
-)
+#' @rdname use_github_labels
+#' @export
+gh_labels <- function() {
```

Comment:
I don't love the current colours, but I find the new colours ugly 😞

Reply:
I reverted the changes to existing colours.